### DISCUSSION TITLE
"Cookies not supported" when trying to put stuff into the cart on Safari (iOS)

### DISCUSSION THREAD

**Discussion by v0tti** [pretix/pretix#3578] [Q&A] [answered]
Hey all,
somehow Safari on iOS won’t let users put tickets in their cart on the Pretix installation I manage. If I try to put tickets in the cart I get a Pretix error page stating that "Cookies are not supported". Other browsers work fine.
At first I thought it might be a problem with iOS blocking third party cookies so I tried different settings for the `cookie_domain` configuration parameter, but that didn't change anything.
This shop is experiencing the problem if anyone wants to try themself: https://tickets.fem.tu-ilmenau.de/fem/wf23/.

Any help would be much appreciated!

**Comment by raphaelm** [comment, marked as answer]
I notice something weird with your server, it sends:

```
set-cookie: pretix_csrftoken=3qDB9djfXzW4BcGeT6jBBC20FB5wKvjSrumK8nj6YqE50ltfi9GUy358Xb7HpgEQ;
    expires=Fri; Secure,06 Sep 2024 07:45:33 GMT; Max-Age=31449600; Path=/; SameSite=None; Secure; Secure
```

A freshly installed pretix that follows the installation documentation only sends the `Secure` flag ones. Is there anything in your webserver that might cause this? I haven't verified it, but I wouldn't be surprised if this is what throws Safari off

**Reply by raphaelm** [reply]
I'm only now noting the `expires=Fri; Secure,06 Sep 2024 07:45:33 GMT` which is even more broken. Something is trying to make this cookie very secure :D

**Reply by v0tti** [reply]
Interesting.
I do not think it is the NGINX serving the Pretix directly but rather the external reverse proxy sitting in front of it. I will try to investigate. Thanks for pointing this out!

**Reply by v0tti** [reply]
So yes, this was a problem with a weirdly configured reverse proxy which is fixed now.
Thanks again!
